packages/dcos-log/buildinfo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "docker": "dcos/dcos-builder:dcos-log_dockerdir-latest",
```

Comment:
magic of custom containers. This builder will use my container to build https://github.com/darkonie/dcos/blob/beb2b1242840327ccd89112b1a03cdf49e8e9265/packages/dcos-log/docker/Dockerfile